tests/golden_tests/readback_lnet/bad_net1.hvm

```diff
@@ -1,2 +1,2 @@
 (a b)
-& {3 c b} ~ (c a)
\ No newline at end of file
+& {2 c b} ~ (c a)
```

Comment:
Does hvm-core completely ignore the value of labels? As in, it doesn't ever look at the meaning of any specific values, just compares them between nodes.

Reply:
correct